mapbox/dr-ui
# Fix padding and background on NumberedCodeSnippet

This PR makes adjustments to the padding and background on NumberedCodeSnippet to prevent adding padding and border effect that was caused by nesting of the element and docs-prose automatically adding styles to the NumberedCodeSnippet:


Status | Image
---|---
Current | ![image](https://user-images.githubusercontent.com/2180540/78388791-9c495680-75af-11ea-9d0b-42e26f67077c.png)
Proposed | ![image](https://user-images.githubusercontent.com/2180540/78393375-a40cf900-75b7-11ea-8ca5-45a1a98af64c.png)


## How to test

1. Open the test cases app to confirm that the component looks correct:
    - /ToggleableCodeBlock
    - /ContextlessAndroidActivityToggle
    - /NumberedCodeSnippet
    - /ContextlessIosViewControllerToggle

## QA checklist



- [ ] Accessibility score in [Chrome DevTools Audit/Lighthouse](https://developers.google.com/web/tools/lighthouse#devtools) is 100% with Lighthouse version: `5.7.0`.
    - Score should be 91 due to https://github.com/mapbox/dr-ui/issues/270

Open the test cases app locally on:
